Dockerfile

```diff
@@ -377,8 +377,6 @@ RUN --mount=from=rootlesskit-src,src=/usr/src/rootlesskit,rw \
   export CGO_ENABLED=$([ "$DOCKER_STATIC" = "1" ] && echo "0" || echo "1")
   xx-go build -o /build/rootlesskit -ldflags="$([ "$DOCKER_STATIC" != "1" ] && echo "-linkmode=external")" ./cmd/rootlesskit
   xx-verify $([ "$DOCKER_STATIC" = "1" ] && echo "--static") /build/rootlesskit
-  xx-go build -o /build/rootlesskit-docker-proxy -ldflags="$([ "$DOCKER_STATIC" != "1" ] && echo "-linkmode=external")" ./cmd/rootlesskit-docker-proxy
```

Comment:
🎉 nice!

⚠️ looks like we need to make some changes in packaging once this is merged, as it's currently expecting this binary to be part of the packages, e.g. (probably other places as well) https://github.com/docker/docker-ce-packaging/blob/2af20a9161e87b9845ea2f6b3bfc0653c583cb25/deb/common/rules#L88